src/components/LogoutModal/LogoutModal.tsx

```diff
@@ -0,0 +1,45 @@
+import { logoutLocally } from '@camp/data-layer';
+import { messages } from '@camp/messages';
+import type { ModalProps } from '@mantine/core';
+import { Button, Group, Modal, Stack, Text } from '@mantine/core';
+
+import { createTestAttr } from '../../utils/createTestAttr';
```

Comment:
Let's use absolute path for this one also

Reply:
yes,I decided about it but I forget to add.
tnx